sphinx_js/renderers.py

```diff
@@ -171,10 +171,15 @@ def first(list, default):
         used_names = []
         MARKER = object()
 
-        for name, default, type in [(param['name'].split('.')[0],
-                                     param.get('defaultvalue', MARKER),
-                                     param.get('type', {'names': []}))
-                                    for param in doclet.get('params', [])]:
+        for param in doclet.get('params', []):
+            name = param['name'].split('.')[0]
+            default = param.get('defaultvalue', MARKER)
+            type = param.get('type', {'names': []})
+
+            # Add '*' to the parameter name if it's a variadic argument
```

Comment:
> I rendered variadics as a Python star-arg, but on second thought, it'd probably make more sense to change it to an ellipsis.

I agree, if Sphinx is willing to render it reasonably.

Reply:
I changed it to an ASCII ellipsis; I thought about a Unicode one, but that isn't actually valid JavaScript.